torch/_inductor/codegen/triton.py

```diff
@@ -2084,6 +2100,20 @@ def decide_later():
         else:
             other = ""
 
+        """Skip L1 cache if we're (pretty?) sure the data is used only once
+        """
+        skip_l1_cache = not (
+            # in all these cases we're likely to reuse the buffer:
+            self.is_broadcasted(original_index)
```

Comment:
Do we need to skip indirect loads (e.g. an embedding lookup) here?  Or is that covered by another case?